settings.py

```diff
@@ -17,6 +17,10 @@
     }
 }
 
+MIDDLEWARE_CLASSES = (
```

Comment:
(seems fine to keep as long as there was a reason - though maybe a comment would be good)

either way i see you've already removed it which is also fine